spec/features/listing_available_dates_spec.rb

```diff
@@ -0,0 +1,11 @@
+feature 'listing available dates' do
+  scenario 'shows a calendar' do
+    sign_up
+    posting
+    click_link 'Main page'
+    click_link 'whatever'
+    expect(page).to have_selector('table tr', count: 6)
```

Comment:
I would avoid making specific references to the HTML you write. If you want to swap this out with a different calendar, you're going to be stuck :blush: